app/server/Main.hs

```diff
@@ -0,0 +1,7 @@
+module Main where
+
+import Prelude
+import Cardano.Wallet
```

Comment:
I don't know what the coding standards are with the wallet team, maybe this is acceptable, but I really prefer explicit imports for anything but prelude.

Reply:
Fixed